src/derivatives/atlas.md

```diff
@@ -0,0 +1,961 @@
+# Templates and atlases
+
+In the following, we describe how the outcomes of analyses that derive from or
+produce [templates and atlases](../common-principles.md#definitions) are organized
+as BIDS-Derivatives.
+These outcomes typically involve quantitative maps, feature maps, parcellations,
+segmentations, and other knowledge annotations such as landmarks defined with
+respect to individual or template brains that are supported by spaces
+such as surfaces and regular grids (images).
+
+For derivatives with atlases in their provenance corresponding to individual subjects,
+the organization follows the standards for BIDS raw and derivatives.
+The following entities MAY be employed to specify template- and atlas-derived results:
+
+-    [`space-<label>`](../glossary.md#space-entities) is REQUIRED to disambiguate derivatives defined with
+     respect to different [coordinate systems](../appendices/coordinate-systems.md), following the general
+     BIDS-Derivatives specifications.
+-    [`cohort-<label>`](../glossary.md#cohort-entities) is REQUIRED to disambiguate derivatives defined with
```

Comment:
This might already do it, but make sure that `cohort-<label>` MUST NOT be used if neither `tpl` nor `space` are used.

Reply:
Is this a logical conditional that should be validated by the spec? The text above this makes it embedded within tpl:

```Text
<pipeline_name>/
    tpl-<label>/
        [cohort-<label>/]
           [<datatype>/]
               tpl-<label>_<source_entities>[_cohort-<label>][_atlas-<label>][seg-<label>][_scale-<label>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.<extension>
```

Reply:
Addressed with https://github.com/bids-standard/bids-specification/pull/1856/commits/c77df1048b6e17805cc5ba9d19db91286b77e88b